registry/storage/paths.go

```diff
@@ -23,25 +23,25 @@ const (
 //
 // The path layout in the storage backend is roughly as follows:
 //
-//		<root>/v2
```

Comment:
All these formatting changes are auto-generated by the go formatter when saving the file.